EIPS/eip-4521.md

```diff
@@ -0,0 +1,62 @@
+---
+eip: 4521
+title: 721/20-compatible transfer
```

Comment:
```suggestion
title: Add `transfer` function to NFT standards
```
This should also work with both 721 and 1155, correct?

Reply:
1155 would be more complicated since it would require the insertion of another argument (since it has both `id` and `amount`), whereas 721s and 20s have the same number of arguments and congruent data types, though 20's uint256 is for the amount of tokens, whereas 721's is for `tokenId`

Reply:
I see, in that case I still think the title needs to be revised to be more clear.

Reply:
I'll go ahead an merge this, but please revise the title before moving to review :)